bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/api/AgsHash.scala

```diff
@@ -56,18 +56,20 @@ object AgsHash {
       c.cc.## +=: c.iq.## +=: c.sb.## +=: buf
     }
 
-    // Base Position
+    // Asterism
     Option(ctx.getTargets).foreach { t =>
       val time = Some(new java.lang.Long(when)).asGeminiOpt
       val asterism = t.getAsterism
 
       def toData(coord: GemOption[java.lang.Double]): Int =
         coord.asScalaOpt.map(_.doubleValue).##
 
-      val ra  = toData(asterism.getRaDegrees(time))
-      val dec = toData(asterism.getDecDegrees(time))
+      asterism.allSpTargets.map { sp =>
+        val ra  = toData(sp.getRaDegrees(time))
+        val dec = toData(sp.getDecDegrees(time))
+        ra +=: dec +=: buf
```

Comment:
It appends an element to mutable `ListBuffer`.  The individual hash values for the various fields of interest are added to a list which is then hashed to a single value by `MurmurHash3`.